<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #fffbe7, stop:1 #e0c097);
    font-family: 'Georgia', 'Times New Roman', serif;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>ðŸ“š Welcome to the Library System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 22px;
font-weight: bold;
color: #4e342e;
margin-top: 18px;
margin-bottom: 12px;
font-family: 'Georgia', 'Times New Roman', serif;
      </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>User Login</string>
     </property>
     <property name="styleSheet">
      <string notr="true">
QGroupBox {
    font-size: 16px;
    color: #4e342e;
    border: 2px solid #a1887f;
    border-radius: 10px;
    margin-top: 10px;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QGroupBox:title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}
      </string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
font-size: 15px;
color: #6d4c41;
font-family: 'Georgia', 'Times New Roman', serif;
         </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName">
        <property name="styleSheet">
         <string notr="true">
QLineEdit {
    border: 2px solid #a1887f;
    border-radius: 10px;
    padding: 6px 14px;
    font-size: 15px;
    background: #fffbe7;
    font-family: 'Georgia', 'Times New Roman', serif;
}
         </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>Email:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
font-size: 15px;
color: #6d4c41;
font-family: 'Georgia', 'Times New Roman', serif;
         </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="styleSheet">
         <string notr="true">
QLineEdit {
    border: 2px solid #a1887f;
    border-radius: 10px;
    padding: 6px 14px;
    font-size: 15px;
    background: #fffbe7;
    font-family: 'Georgia', 'Times New Roman', serif;
}
         </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelRole">
        <property name="text">
         <string>Role:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
font-size: 15px;
color: #6d4c41;
font-family: 'Georgia', 'Times New Roman', serif;
         </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxRole">
        <property name="styleSheet">
         <string notr="true">
QComboBox {
    border: 2px solid #a1887f;
    border-radius: 10px;
    padding: 6px 14px;
    font-size: 15px;
    background: #fffbe7;
    font-family: 'Georgia', 'Times New Roman', serif;
}
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
     <property name="styleSheet">
      <string notr="true">
QDialogButtonBox {
    margin-top: 10px;
}
QPushButton {
    background-color: #4e342e;
    color: #fffbe7;
    border-radius: 10px;
    padding: 8px 24px;
    font-size: 15px;
    font-weight: bold;
    font-family: 'Georgia', 'Times New Roman', serif;
}
QPushButton:hover {
    background-color: #3e2723;
}
      </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
